fastcrypto-tbls/src/tbls.rs

```diff
@@ -8,14 +8,15 @@ use std::borrow::Borrow;
 
 use crate::dl_verification::{batch_coefficients, get_random_scalars};
 use crate::polynomial::Poly;
-use crate::types::IndexedValue;
+use crate::types::{IndexedValue, UnindexedValues};
 use fastcrypto::error::{FastCryptoError, FastCryptoResult};
 use fastcrypto::groups::{GroupElement, HashToGroupElement, MultiScalarMul, Scalar};
 use fastcrypto::traits::AllowedRng;
 use itertools::Itertools;
 
 pub type Share<S> = IndexedValue<S>;
 pub type PartialSignature<S> = IndexedValue<S>;
+pub type CompactPartialSignatures<S> = UnindexedValues<S>;
```

Comment:
optional suggestion, I would maybe call this `UnindexedPartialSignatures` just so we aren't introducing a second term "Compact" to mean the ~same thing?